host_vars/mongodb.opencraft.hosting/public.yml

```diff
@@ -0,0 +1,11 @@
+mongodb_user_admin_name: "{{ MONGODB_USER_ADMIN_NAME }}"
+mongodb_user_admin_password: "{{ MONGODB_USER_ADMIN_PASSWORD }}"
+
+mongodb_root_admin_name: "{{ MONGODB_ROOT_ADMIN_NAME }}"
+mongodb_root_admin_password: "{{ MONGODB_ROOT_ADMIN_PASSWORD }}"
+
+mongodb_keyfile_content: "{{ MONGODB_KEYFILE_CONTENT }}"
+
+TARSNAP_KEY: "{{ MONGODB_TARSNAP_KEY }}"
+SANITY_CHECK_SNITCH: "{{ MONGODB_SANITY_CHECK_SNITCH }}"
+TARSNAP_BACKUP_SNITCH: "{{ MONGODB_TARSNAP_BACKUP_SNITCH }}"
```

Comment:
Can we keep the public repo free of references to the individual servers, to keep it generic? Is it possible for example to move the `host_vars` directory to the private repository, which is the one with the knowledge of the different hosts? Or, if not, a dict in the private vars containing the values for each host, and just use a sample default in the (public) `group_vars`?